### PR TITLE
update non street classes

### PR DESCRIPTION
**Pull request by ZenVega**

### CODE REVIEW

File: common.js
Comment (Bjorn248) [inline common.js:1823]: Remove `all` here. Try not to have `all` and specific years.

File: common.js
Comment (Bjorn248) [inline common.js:1809]: Remove `all`

File: common.js
Comment (Bjorn248) [inline common.js:1801]: Remove `all`

File: common.js
Comment (Bjorn248) [inline common.js:1733]: Remove `all`

File: common.js
Comment (Bjorn248) [inline common.js:1648]: Remove `all`

File: common.js
Comment (Bjorn248) [inline common.js:1844]: Remove `all` here and just make sure that every specific year has `stu` listed.

File: common.js
Comment (Bjorn248) [inline common.js:1858]: Remove `all` and make sure that every year has `es`

File: common.js
Comment (Bjorn248) [inline common.js:1866]: Remove `all`, make sure every year has `es`

File: common.js
Comment (Bjorn248) [inline common.js:1940]: Remove `all`

File: common.js
Comment (Bjorn248) [inline common.js:1967]: Remove empty line

**Review by Bjorn248 — CHANGES_REQUESTED** [review]
Should be ready to merge after some adjustments

File: common.js
Comment (ZenVega) [reply common.js]: ok, i think i got it

**Review by Bjorn248 — CHANGES_REQUESTED** [review]
Found a syntax error

**Comment by Bjorn248** [comment]
@ZenVega would it be possible for you to rebase off of master and resolve the merge conflicts? Or perhaps just copy your toyota block and paste it into the file after you've pulled the latest code? Thanks.

**Comment by ZenVega** [comment]
@Bjorn248  hey sorry, I don't really get what u mean. why did this conflict happen anyway? do you mean I shall make a PR with the non-Master branch?

**Comment by Bjorn248** [comment]
@ZenVega This conflict happened because the upstream code (the code hosted on github) was updated after you initially cloned it. You will either have to locally resolve the merge conflicts by doing a `git fetch` and `git pull`, or by cloning the repo again and pasting your changes into a more updated version of the repo. I think doing the former is a good exercise in learning git. Let me know if you need any more guidance, I can try to point you in the right direction.

**Comment by ZenVega** [comment]
hmm... no, it seems I'm not getting it. so I git fetch the original(your repo)?
then git pull. (then it says everything is up to date) and pushing again doesn't seem to solve the problem

**Comment by Bjorn248** [comment]
@ZenVega try following this guide https://www.youtube.com/watch?v=__cR7uPBOIk. Let me know if this works out 😃

**Comment by ZenVega** [comment]
oh man, I messed up my repository big time so I decided to solve it here in the commit.

**Comment by Bjorn248** [comment]
You're deleting the rest of the files in the repository, so I definitely cannot merge this. If you want to you can try copying your object to a local file, deleting your local copy of the repository, deleting your fork on github, reforking, cloning it from github again, and pasting your new object in and then opening a new PR? Kind of the "nuke it" option but it might be the easiest way to get to the finish line from here.

**Pull request closed without merging**
